app/layout.tsx

```diff
@@ -1,6 +1,7 @@
 import Script from "next/script";
 import React from "react";
 import { Metadata } from "next";
+import "./globals.css";
```

Comment:
This works well so feature-wise, approved.

Two things with this file though. Can it be scss instead of plain css? That way you might be able to use css vars from the DS for the min-width breakpoint values. If not possible because of the order the files load, no big deal.

My bigger concern are all default rules in the `global.css` files. They came from the default nextjs starter kit and they weren't used before so you can clean up and remove the default and only keep the new rules you added.

Reply:
All unused defaults removed– I think they were getting overridden by the DS Provider so I didn't notice. Also, I'm not sure the values I'm using here are actually vars in the DS since these are the new breakpoints... am I missing a set of spacing variables somewhere

Reply:
Yes, they're not in the DS. Sorry, I meant from the internally defined breakpoints file.

Reply:
Actually, pulling in the js won't work and you could create CSS variables at the root level in `global.css`. Regardless, you'll have to update it whenever the new breakpoint values come out. I don't expect they'll come out until v4 of the DS which might be a while. So for now this is fine.

Reply:
I'm a lot less familiar with Sass than standard CSS but it seems like I would still need to re-declare those breakpoint variables for Sass (like in a `_variables.scss` file or just at the top of `globals`), and even if this wasn't the case I'm not sure it feels worth it to add the whole package just for this use. Is that implementation what you were picturing ?